enhancements/service-catalog/service-catalog-and-broker-removal.md

```diff
@@ -87,69 +84,103 @@ cluster-svcat-controller-manager-operator, with a [Job](https://kubernetes.io/do
+cluster. The Job that will delete the Service Catalog operators and their
+associated items.
+
+- cluster-svcat-apiserver-operator
+  - Role named `prometheus-k8s in `openshift-service-catalog-apiserver-operator`
+  - RoleBinding named `prometheus-k8s in `openshift-service-catalog-apiserver-operator`
+  - ServiceMonitor in `openshift-service-catalog-apiserver-operator`
+  - Namespace `openshift-service-catalog-apiserver-operator` (not sure
+    if we can do this from the Job)
+  - ServiceCatalogAPIServer (cr)
+  - ConfigMap named `openshift-service-catalog-apiserver-operator-config`
+  - Service named `metrics`
+  - ClusterRoleBinding for `openshift-service-catalog-apiserver-operator` ServiceAccount
+  - ServiceAccount `openshift-service-catalog-apiserver-operator`
+  - Deployment named `openshift-service-catalog-apiserver-operator`
+  - ClusterOperator named `service-catalog-apiserver`
+- cluster-svcat-controller-manager-operator
+  - Role named `prometheus-k8s in `openshift-service-catalog-controller-manager-operator`
+  - RoleBinding named `prometheus-k8s in `openshift-service-catalog-controller-manager-operator`
+  - ServiceMonitor in `openshift-service-catalog-controller-manager-operator`
+  - Namespace `openshift-service-catalog-controller-manager-operator` (not sure
+    if we can do this from the Job)
+  - ServiceCatalogControllerManager (cr)
+  - ConfigMap named `openshift-service-catalog-controller-manager-operator-config`
+  - Service named `metrics`
+  - ClusterRoleBinding for `openshift-service-catalog-controller-manager-operator` ServiceAccount
+  - ServiceAccount `openshift-service-catalog-controller-manager-operator`
+  - Deployment named `openshift-service-catalog-controller-manager-operator`
+  - ClusterOperator named `service-catalog-controller-manager`
+
+The Job will NOT touch the Service Catalog resources. These resources will
+remain if the admin marked the Service Catalog CRs as `Unmanaged` meaning they
+will self manage the Service Catalog relieving the operators of their duty.
+
+The Service Catalog operators would remain in the release payload but the
 Service Catalog itself would be removed.
 
-There are a few alternatives that are discussed in the Alternatives section of
-this proposal, please see below.
-
 #### Story 2
 
-Implement a mechanism that ensures Service Brokers are not present after
-an upgrade to OpenShift 4.4.
-
-Create a new CVO managed operator with a Job definition as in Story 1 to remove
-the Ansible Service Broker and Template Service Broker. The new CVO operator is
-needed because the Ansible Service Broker operator and the Template Service
-Broker operator are both OLM managed operators which may or may not be present
-during cluster upgrades.
-
-NOTE: this would add a new item to the OpenShift 4.4 release payload.
-
-There is also an alternative to this approach in the Alternatives section below.
+Keep the Service Catalog _Operators_ in the OpenShift 4.4 release payload.
```

Comment:
Keep the operators as in "keep the operator images and yamls necessary to deploy the operators" or "replace the existing svccat operators' YAMLs with the Job image and YAML described above that deletes the operators"?

Should this be _operands_?

Reply:
Service Catalog itself is the operand. It has been removed from productization already. So what I'm trying to say here is that the Service Catalog operator yamls and a job image will be in the payload but the operator code will go away.

I can update this to be a little more clear since Shawn had a similar question.